wfe/web-front-end.go

```diff
@@ -708,7 +708,11 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 
 		cert, err := wfe.SA.GetCertificateByShortSerial(serial)
 		if err != nil {
-			wfe.sendError(response, "Not found", err, http.StatusNotFound)
+			if strings.HasPrefix(err.Error(), "gorp: multiple rows returned") {
```

Comment:
Rather than depend on the exact error string from gorp, how about doing a multiple-row select query (with limit 2, just to be paranoid about large results) and failing if more than one row is returned?